### PR TITLE
Fix travis

### PR DESCRIPTION
**Pull request by frett**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -35,22 +35,33 @@ cache:
...
     - $HOME/.gradle/wrapper
 
-# Emulator Management: Create, Start and Wait
-before_script:
-  - echo no | android create avd --force -n test -t $ANDROID_TARGET --abi $ANDROID_ABI
-  - emulator -avd test -no-skin -no-audio -no-window &
+before_install:
+  # Install SDK license so Android Gradle plugin can install deps.
+  - mkdir "$ANDROID_HOME/licenses" || true
+  - echo "d56f5187479451eabf01fb78af6dfcb131a6481e" > "$ANDROID_HOME/licenses/android-sdk-license"
```
Comment (ryancarlson) [inline .travis.yml]: Does this need to be an encrypted variable?

Comment (frett) [reply .travis.yml]: it's the same for everyone

Comment (ryancarlson) [reply .travis.yml]: 👍

**Comment by frett** [comment]
hmm, it looks like one of our tests is actually failing on API 24. I'll deal with that in a different PR.

**Pull request merged**